src/test/ui/print_type_sizes/nullable.stdout

```diff
@@ -0,0 +1,24 @@
+print-type-size type: `IndirectNonZero<u32>` overall bytes: 40 align: 8
+print-type-size    field .pre bytes: 8
+print-type-size    field .nested bytes: 24
+print-type-size    field .post bytes: 8
+print-type-size type: `MyOption<IndirectNonZero<u32>>` overall bytes: 40 align: 8
+print-type-size    variant Some exact bytes: 40
+print-type-size        field .0 bytes: 40
```

Comment:
Some nitpicks:
- Do the "overall" and "exact" words contribute anything here? Can they be removed without loss of information?
- Inconsistent colon placement: in a different place for "type" compared to  "field" and "variant".
- Perhaps the "bytes" should appear after the numbers, on the right-hand side. 
- Field names should have backticks around them, like type names.
- A comma before the alignment would help readability.
- "alignment" is probably better than "align".
- First indent is 3 spaces, second indent is 4 spaces. Use 4 (or 2) consistently.

E.g.:
```
print-type-size type `MyOption<IndirectNonZero<u32>>`: 40 bytes, alignment: 8 bytes
print-type-size     variant `Some`: 40 bytes
print-type-size         field .0: 40 bytes
```